### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD032.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#35] [General]
# https://blog.3place.cn/3place/fsd/FSD032.html

https://blog.3place.cn/3place/fsd/FSD032.html

**Comment by picklecai** [comment]
月亮虽不圆满，却发出明亮的光，
升上天空，把她的柔辉遍照四方；
帐篷的幻影消逝，篝火发出青光。

这个青光，很有气氛。

**Comment by picklecai** [comment]
请你放平你的骑士，眼睛一瞬，
他就恢复他的生命，
他到神话之国来求生。

浮士德
（触到地面。）
她在哪里？——

荷蒙库路斯

我们不知道，
但在这里总可以问到。
你可以赶快，趁天色未明，
前去各堆篝火边找她：
你有胆量去寻找“母亲”，
别的还有什么害怕。

刚一触地，浮士德就能说话了？好神奇的疗法。而且是让他自救。

**Comment by picklecai** [comment]
我们心底里虽也不大守规矩，
我却觉得老古董过于露骨；
必须用最新的精神加以控制，
给它加上各种时式的粉饰……

古代的人类，比新世纪的魔鬼还要露骨。
露骨这个词用得太感情化了。应该说，原始人类，比后世的魔鬼更少“文明”的束缚，别说后世的人类了。

**Comment by picklecai** [comment]
梅非斯特

（坐到两只人面狮之间。）
在这里厮混，多么容易而高兴，
他们说的，我全能了解。

梅菲斯特是不是想说，人类太狡诈，不如这些妖实在赤城？

**Comment by picklecai** [comment]
人面狮
谈谈你自己，这就等于出谜。
请分析一下你的内心深处：
“对于善人和恶人同样必需，
对善人是胸甲，练习苦行的击剑，
对恶人是搭档，进行胡闹的捣乱，
两者都只为博得宙斯的喜欢。”

这句话的来源，是苏格拉底的“认识你自己”吧？认识自己，是最大的困难，所以，这就是一个谜。

**Comment by picklecai** [comment]
梅非斯特
从前你会把他们赶得精光，
现在倒像是对你有利；
因为在寻爱人的地方，
碰到妖怪也欢迎之至。

只要你有一个目标，一切就皆可利用。

**Comment by picklecai** [comment]
你要干什么事情？
为什么显得心神不定？
你要去哪里？尽管请便！……
我看出，那边的合唱队员
使你变成歪脖。
不要勉强，
去吧！看看许多漂亮的面庞。

变成歪脖，哈哈哈，这个词太形象了。

**Comment by picklecai** [comment]
我们就这样规定日月岁时。
我们坐在金字塔前，
阅尽诸民族的兴亡；
战争、和平、洪水泛滥——
都像若无其事一样。

好一股镇定的力量。

**Comment by picklecai** [comment]
摇曳吧，你萧萧的芦苇！
轻轻透气吧，蒹葭姊妹，
沙沙作响吧，袅袅的柳丝，
低语吧，战栗的白杨树枝，

非河神不能如此。但我们的河神，只顾着受人间的香火，无暇顾及座下生灵。

**Comment by picklecai** [comment]
我应该对此感到满足，
应该在这里一饱眼福，
可是我总有更大的要求。

此时的目标，只是海伦。

**Comment by picklecai** [comment]
其中有一只与众不同，
大摇大摆，昂首挺胸，
穿过鹅群而迅速前进；
把羽毛张得蓬蓬松松，
像白波一样在水面波动，
一直闯入那一处圣境……

这是宙斯吧？

**Comment by picklecai** [comment]
弟子们后来总是各行其是，
就像没受过教育一样。

最清醒的老师：菩提祖师，以及刻戎。

**Comment by picklecai** [comment]
妇女的美丽有什么稀奇，
往往不过是呆板的形象；
只有洋溢着快乐的生气，
我才能对她加以颂扬。
丽质只可以自己欣赏；
优美才使人无法抵抗，
像我当初背过的海伦。

美丽在于活力。

**Comment by picklecai** [comment]
因为，如果没有魔女，
哪个恶魔愿做恶魔！

哈哈哈！梅菲斯特太可爱了。

**Comment by picklecai** [comment]
瓦尔普吉斯之夜，和第一部用了同一个名字。但是第一部中梅菲斯特熟悉得像个主人，第二部中梅菲斯特却成了异邦客人。

**Reply by picklecai** [reply]
说：第二部是古典的瓦尔普吉斯之夜，指古代希腊的群魔大会，地点在忒萨利亚平原，时间为6月6日（恺撒战胜庞培之日）前夕。
“古典的”是跟第一部“浪漫的”瓦尔普吉斯之夜（北方德意志的）相对而言。

**Comment by picklecai** [comment]
我向可爱的蒙面人伸手，
抓到的却使我心惊胆战……
如果能够稍许持久，
我倒也甘心受她们欺骗。

梅菲斯特说的就是：假如对方能骗我一辈子，那么我甘愿被骗。
魔鬼也吃软。

**Comment by picklecai** [comment]
梅非斯特
这种事还是听凭自己。
因为在幽灵得势的地境，
哲学家也会受到欢迎。
为了让人欣赏他的本领，
会造出一打新的幽灵。

看来梅菲斯特对所谓“哲学家”也是腹诽。

**Comment by picklecai** [comment]
你不走错路，就不会懂得事理！
你要成长，务须靠你自己。

魔鬼都知道：不走错路，就不会懂事。
或者说，魔鬼更知道这个道理。因为他之所以被称为魔鬼，大概就是这不怕犯错的勇气。

**Comment by picklecai** [comment]
阿那克萨戈拉（前500-前428，和曾子他们同龄人）：

阿那克萨戈拉出生于爱奥尼亚的克拉佐美尼，古希腊哲学家、原子唯物论的思想先驱，克拉左美尼人。他是米利都学派。
在雅典人战胜了波斯人之后，他被老师带到了雅典。他是第一个把哲学介绍给雅典人的。由于他否认天体是神圣的，因此被控亵渎神圣，幸亏伯里克利的调解才得活命。公元前 464年来到雅典，居住了30年。他是雅典奴隶主民主派领袖伯里克利的朋友、老师和政治上的积极支持者。他是著名的自然科学家，认为太阳是一团炽热的物质，月亮和地球一样也有山谷和居民，陨石是从太阳掉下来的石头，雷由云彩的撞击而产生，闪电是云与云之间摩擦的结果。由于这些违反传统宗教和神话的主张，被人攻击为宣传邪说，以“不敬神”的罪名被驱逐出雅典。后来他回到伊奥尼亚，隐居于朗普萨柯。

**Comment by picklecai** [comment]
泰勒斯

我不劝你；
跟小人一起，只能成小事，
跟大人，小事可成大功。

这是鸡头凤尾的争论。歌德的泰勒斯，持凤尾论。

**Comment by picklecai** [comment]
梅非斯特
怎么不怕！——一看就使我惊骇。
我虽傲慢，可也不得不承认：
从未见过这样的丑人，
比阿尔戎涅还要厉害……

梅菲斯特是个外貌协会的哦，今天多次表达了不喜欢长得丑的。

**Comment by picklecai** [comment]
尤诺、帕拉斯、维纳斯

就是天后赫拉、雅典娜和维纳斯。

**Comment by picklecai** [comment]
你们本该在那种地方栖身，
那里，荣华与艺术地位相等，
那里，每天以一种跑步速度，
大理石块活现出英雄人物。

这是文艺复兴时期的意大利吧？

**Comment by picklecai** [comment]
梅菲斯特为什么要变成福耳库阿斯的样子？

**Comment by picklecai** [comment]
现在请从你的夜空，
静看这一片波涛汹涌，
千点万点闪耀的柔光，
照临这些熙熙攘攘
从水波里升起的群妖。

去掉“群妖”两个字的话，波澜壮阔，还是挺舒服的感觉。
有了这两个字，一下子密集恐惧症犯了。

**Comment by picklecai** [comment]
温柔的路娜，请你
慈祥地永留天际，
别让黑夜消残，
白昼把我们驱赶。

刚刚反应过来，路娜就是月亮。

**Comment by picklecai** [comment]
涅柔斯，貌似是波塞冬的前任。

**Comment by picklecai** [comment]
他的脑筋顽固，
脾气暴躁，不易对付。
这个乖僻人对人类全体
总觉得不遂他的心意。
但他知道未来的事情，
因此人人都对他尊敬，
大家尊重他的地位；
他也对人广开善门。

这个描述，多像个老父亲

**Comment by picklecai** [comment]
这些众生，拚命想修成神仙，
却注定他们永远故我依然。
我自古以来就能安居如神，
却总禁不住要去嘉惠善人；
可是最后将结果加以检验，
就像我完全没有进过忠言。

解释了为什么涅柔斯会脾气暴躁，因为建议不能被采纳。爱使他善，爱使他怒。

**Comment by picklecai** [comment]
尤利斯就是奥德修斯。

**Comment by picklecai** [comment]
普洛透斯（Πρωτεύς / Proteus）是希腊神话中的一个早期海神，荷马所称的“海洋老人”之一。他的名字可能有“最初”的含义，因为希腊文"protogonos"表示“最早出世的”。最初并没有提及他的家系，直到后来的神话学者才将他归为奥林匹斯神波塞冬的后代，或者是涅柔斯和多里斯之子，或是俄刻阿诺斯同一个放牧海豹的奈阿得斯所生。他有预知未来的能力，但他经常变化外形使人无法捉到他：他只向逮到他的人预言未来。

**Comment by picklecai** [comment]
泰勒斯

这正是世人之所贪求，
钱币的价值在于生锈。

有了钱就要藏起来、存起来，所以泰勒斯如此说。

**Comment by picklecai** [comment]
普洛透斯
（现出高贵的形体。）
处世的手腕你还没有遗忘。

泰勒斯
你到如今还喜爱变形改相。
（将荷蒙库路斯显露出来。）

泰勒斯引诱普洛透斯出来的场景，和他租榨橄榄机的故事，真的吻合。这个人，想入世时可以非常入世。

**Comment by picklecai** [comment]
现在不需想得过分，
你要先到大海里去发轫！
在那里先从小处开场，
乐于吸收极微的养分，
这样就能渐渐成长，
使你自己臻于更高的化境。

生命起源于海洋。以海之大，不捐细流，可以吸收极细微的养分。

**Comment by picklecai** [comment]
比起太阳的生命之圣光，
死板的制作只是儿戏。

一切艺术，不过是对自然的拙劣模仿。

**Comment by picklecai** [comment]
不要急于爬往上层，
因为等到你一变成人，
你的发展就到此为止。

目标不要马上实现，过程实在值得享受。
